aws/resource_aws_workspaces_directory_test.go

```diff
@@ -658,6 +804,50 @@ resource "aws_workspaces_directory" "main" {
 `, rName))
 }
 
+func testAccWorkspacesDirectoryConfig_workspaceCreationProperties_customSecurityGroupId_defaultOu_Absent(rName string) string {
+	return composeConfig(
+		testAccAwsWorkspacesDirectoryConfig_Prerequisites(rName),
+		fmt.Sprintf(`
+resource "aws_security_group" "test" {
```

Comment:
extraneous as its not used in this config

Reply:
Sharp eyes 🦅  Thank you, I'll drop it.